src/crimson/os/cyanstore/cyan_store.cc

```diff
@@ -276,6 +276,20 @@ CyanStore::omap_get_values(
     true, values);
 }
 
+seastar::future<ceph::bufferlist>
+CyanStore::omap_get_header(
+    CollectionRef ch,
+    const ghobject_t& oid
+  ) {
```

Comment:
i'd suggest continue using the coding style in this source file instead of creating a new one. so could you reformat the code like
```c++
seastar::future<ceph::bufferlist>
CyanStore::omap_get_header(CollectionRef ch, const ghobject_t& oid)
{
  // ...
}
```
?